rct229/rulesets/ashrae9012019/section4/section4rule1.py

```diff
@@ -220,10 +220,15 @@ def get_calc_vals(self, context, data=None):
                     "heating_schedule_matched": heating_schedule_matched,
                 }
 
-            def manual_check_required(self, context, calc_vals=None, data=None):
+            def rule_check(self, context, calc_vals=None, data=None):
                 cooling_schedule_matched = calc_vals["cooling_schedule_matched"]
                 heating_schedule_matched = calc_vals["heating_schedule_matched"]
-                return not (cooling_schedule_matched and heating_schedule_matched)
 
-            def rule_check(self, context, calc_vals=None, data=None):
-                return True
+                return cooling_schedule_matched and heating_schedule_matched
+
+            def get_fail_msg(self, context, calc_vals=None, data=None):
+
+                return (
+                    "There is a temperature schedule mismatch between the baseline and proposed RMDs. "
+                    "Fail unless table g3.1 #4 baseline column exception #s 1 and/or 2 are applicable"
```

Comment:
should Table G3.1 be in lower case? Just a minor comment.

Reply:
Good catch! Updated.